CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed issue with dependency graph failing wait for step dependencies to
```

Comment:
`failing wait` -> `failing to wait` I think.

Reply:
oh, you fixed it.

Reply:
Fixed